styles/styles.css

```diff
@@ -156,48 +145,229 @@ a:hover {
+    --color-button-border-primary-outline-disabled: rgb(from var(--spectrum-white) r g b / 17%);
+    --color-button-background-primary-outline-hover: rgb(from var(--spectrum-white) r g b / 11%);
+    --color-button-text-primary-outline: rgb(from var(--spectrum-white) r g b / 85%);
+    --color-button-text-primary-outline-hover: rgb(from var(--spectrum-white) r g b / 94%);
+    --color-button-text-primary-outline-disabled: rgb(from var(--spectrum-white) r g b / 21%);
+  }
+}
+
+.button--static-black {
+  --color-button-border-static-black: rgb(from var(--spectrum-black) r g b / 84%);
+  --color-button-border-static-black-hover: rgb(from var(--spectrum-black) r g b / 93%);
+  --color-button-border-static-black-disabled: rgb(from var(--spectrum-black) r g b / 15%);
+  --color-button-background-static-black-hover: rgb(from var(--spectrum-black) r g b / 9%);
+  --color-button-text-static-black: rgb(from var(--spectrum-black) r g b / 84%);
+  --color-button-text-static-black-hover: rgb(from var(--spectrum-black) r g b / 93%);
+  --color-button-text-static-black-disabled: rgb(from var(--spectrum-black) r g b / 22%);
+
+  border: 2px solid var(--color-button-border-static-black);
+  color: var(--color-button-text-static-black);
+
+  &:hover {
+    border-color: var(--color-button-border-static-black-hover);
+    background-color: var(--color-button-background-static-black-hover);
+    color: var(--color-button-text-static-black-hover);
+  }
+  
+  &:disabled,
+  &:disabled:hover {
+    border-color: var(--color-button-border-static-black-disabled);
+    color: var(--color-button-text-static-black-disabled);
+  }
+}
+
+.button--static-white {
+  --color-button-border-static-white: rgb(from var(--spectrum-white) r g b / 85%);
+  --color-button-border-static-white-hover: rgb(from var(--spectrum-white) r g b / 94%);
+  --color-button-border-static-white-disabled: rgb(from var(--spectrum-white) r g b / 17%);
+  --color-button-background-static-white-hover: rgb(from var(--spectrum-white) r g b / 11%);
+  --color-button-text-static-white: rgb(from var(--spectrum-white) r g b / 85%);
+  --color-button-text-static-white-hover: rgb(from var(--spectrum-white) r g b / 94%);
+  --color-button-text-static-white-disabled: rgb(from var(--spectrum-white) r g b / 21%);
+
+  border: 2px solid var(--color-button-border-static-white);
+  color: var(--color-button-text-static-white);
+  
+  &:hover {
+    border-color: var(--color-button-border-static-white-hover);
+    background-color: var(--color-button-background-static-white-hover);
+    color: var(--color-button-text-static-white-hover);
+  }
+  
+  &:disabled,
+  &:disabled:hover {
+    border-color: var(--color-button-border-static-white-disabled);
+    color: var(--color-button-text-static-white-disabled);
+  }
+}
+
+.button--accent {
+  --color-button-background-accent: var(--spectrum-blue-900);
+  --color-button-background-accent-hover: var(--spectrum-blue-1000);
```

Comment:
<img width="799" alt="Screenshot 2025-04-25 at 7 32 04 AM" src="https://github.com/user-attachments/assets/ee6985dd-9ff8-4924-809c-c26d85edf84a" />

In the designs, the hover state for the accent button on light and dark mode is the same. I wondered if this might have been an oversight so I took cues from S2 designs, where on light mode the background darkens.

I corrected the background blues on the light mode and dark mode accent button as well.

Reply:
Nice work!!